CorsixTH/Lua/world.lua

```diff
@@ -67,8 +67,8 @@ function World:World(app)
   self.entity_map = EntityMap(self.map)
 
   -- Time
-  self.hours_per_tick = 2
-  self.tick_rate = 3
+  self.hours_per_tick = 1
+  self.tick_rate = 6
```

Comment:
```suggestion
  self.tick_rate = 3
```
This matches the Normal speed set below.